InvenTree/plugin/builtin/suppliers/mouser.py

```diff
@@ -0,0 +1,49 @@
+"""The MouserPlugin is meant to integrate the Mouser API into Inventree.
+
+This plugin currently only match Mouser barcodes to supplier parts.
+"""
+
+import logging
+
+from django.utils.translation import gettext_lazy as _
+
+from plugin import InvenTreePlugin
+from plugin.base.barcodes.mixins import SupplierBarcodeData
+from plugin.mixins import SettingsMixin, SupplierBarcodeMixin
+
+logger = logging.getLogger('inventree')
+
+
+class MouserPlugin(SupplierBarcodeMixin, SettingsMixin, InvenTreePlugin):
```

Comment:
The `MouserPlugin` is the only one which inherits from the `SettingsMixin` - so the settings don't yet work for the other supplier plugins. Please add `SettingsMixin` to the other ones (e..g digikey / tme / etc)